packages/cloud_firestore/cloud_firestore_web/lib/src/interop/firestore_interop.dart

```diff
@@ -480,7 +479,8 @@ abstract class TimestampJsImpl {
 @anonymous
 @JS()
 abstract class FirestoreError {
-  external String /*|'cancelled'|'unknown'|'invalid-argument'|'deadline-exceeded'|'not-found'|'already-exists'|'permission-denied'|'resource-exhausted'|'failed-precondition'|'aborted'|'out-of-range'|'unimplemented'|'internal'|'unavailable'|'data-loss'|'unauthenticated'*/ get code;
+  external String /*|'cancelled'|'unknown'|'invalid-argument'|'deadline-exceeded'|'not-found'|'already-exists'|'permission-denied'|'resource-exhausted'|'failed-precondition'|'aborted'|'out-of-range'|'unimplemented'|'internal'|'unavailable'|'data-loss'|'unauthenticated'*/
+      get code;
 
   external set code(
       /*|'cancelled'|'unknown'|'invalid-argument'|'deadline-exceeded'|'not-found'|'already-exists'|'permission-denied'|'resource-exhausted'|'failed-precondition'|'aborted'|'out-of-range'|'unimplemented'|'internal'|'unavailable'|'data-loss'|'unauthenticated'*/
```

Comment:
changes in this file are not related to the actual issue this PR is intended to fix, so please revert these

Reply:
Sorry about the unintended changes. Those probably snuck in when I ran the formatter perhaps? Anyways, I updated the branch to only include the intended changes. Please let me know if there is anything else I can do. Thank you for the fast review!